src/dune_pkg/rev_store.ml

```diff
@@ -913,11 +913,7 @@ let content_of_files t files =
 
 let get =
   Fiber_lazy.create (fun () ->
-    let dir =
-      Path.L.relative
-        (Path.of_string (Xdg.cache_dir (Lazy.force Dune_util.xdg)))
-        [ "dune"; "git-repo" ]
-    in
+    let dir = Path.L.relative Dune_cache_storage.Layout.root_dir [ "dune"; "git-repo" ] in
```

Comment:
Does `Dune_cache_storage.Layout.root_dir` return the contents of `$XDG_CACHE_HOME` (or similar) if `$DUNE_CACHE_ROOT` is not set?

Reply:
Yes, it calls `Xdg.cache_dir` which is set from the env var

Reply:
Is this not adding one layer of `dune` too much? I would expect it to return `~/.cache/dune` if the directory was not set thus it would be `~/.cache/dune/dune/git-repo`.